README.md

```diff
@@ -62,6 +61,25 @@ func DoSomething(input string) (string, error) {
 }
 ```
 
+When declaring errors at the package scope, be sure not to use an serrors
```

Comment:
opt: Wording feels off, maybe something like "be sure to use `errors.New` instead of an serrors function, as the..."

Reply:
Reworked the whole paragraph, but hit me with it again if it's still missing the mark. It's tough to unambiguously describe the error declared at the package scope, the error annotation, and the application error in paragraph form.